homeassistant/components/emulated_hue/config.py

```diff
@@ -259,7 +259,4 @@ def _is_state_exposed(self, state: State) -> bool:
         # Expose an entity if the entity's domain is exposed by default and
         # the configuration doesn't explicitly exclude it from being
         # exposed, or if the entity is explicitly exposed
-        if state.domain in self.exposed_domains:
-            return True
-
-        return False
+        return state.domain in self.exposed_domains
```

Comment:
Same here. I won't comment on the following cases as it's repetitive. Left my option about it in a comment above.